### PR TITLE
Make basic Waku v2 examples run again

### PR DESCRIPTION
**Pull request by oskarth**
Regression introduced in https://github.com/status-im/nim-waku/pull/117

Happened due to example not being run on CI.

- Add wakuexample to all target
- Fix example2 to run
- Basic formatting

### CODE REVIEW

**Review by kdeme — COMMENTED** [review]
Sorry, I thought I tested all in #117 but the example slipped through.

File: examples/v2/basic2.nim [hunk 1]
```diff
@@ -31,6 +32,7 @@ proc runBackground(conf: WakuNodeConf) {.async.} =
   let message = cast[seq[byte]]("hello world")
   node.publish(topic, message)
 
-discard runBackground(conf)
+
+discard runBackground()
```
[the remark this answers is not in the document]
Comment (oskarth) [reply examples/v2/basic2.nim:36]: I just want it to run for now. Agree it is an issue and we should clean it up in a bunch of places. Can we address this in a separate PR?

File: examples/v2/basic2.nim [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
+proc runBackground() {.async.} =
+  let
+    conf = WakuNodeConf.load()
+    (extIp, extTcpPort, extUdpPort) = setupNat(conf.nat, clientId,
+      Port(uint16(conf.tcpPort) + conf.portsShift),
+      Port(uint16(conf.udpPort) + conf.portsShift))
+    node = WakuNode.init(conf.nodeKey, conf.libp2pAddress,
+      Port(uint16(conf.tcpPort) + conf.portsShift), extIp, extTcpPort)
+
+  waitFor node.start()
```
Comment (oskarth) [inline examples/v2/basic2.nim:24]:
```suggestion
  await node.start()
```

File: examples/v2/basic2.nim [hunk 3]
```diff
@@ -31,6 +33,7 @@ proc runBackground(conf: WakuNodeConf) {.async.} =
   let message = cast[seq[byte]]("hello world")
   node.publish(topic, message)
 
-discard runBackground(conf)
+
```
Comment (oskarth) [inline examples/v2/basic2.nim:36]:
```suggestion
TODO Await with try/except here
```

File: examples/v2/basic2.nim (hunk 1 above)
[the remark this answers is not in the document]
Comment (kdeme) [reply examples/v2/basic2.nim:37]: sure

File: examples/v2/basic2.nim [hunk 4]
```diff
@@ -31,6 +33,7 @@ proc runBackground(conf: WakuNodeConf) {.async.} =
   let message = cast[seq[byte]]("hello world")
   node.publish(topic, message)
 
-discard runBackground(conf)
+TODO Await with try/except here
```
Comment (oskarth) [inline examples/v2/basic2.nim:36]:
```suggestion
# TODO Await with try/except here
```

File: examples/v2/basic2.nim (hunk 1 above)
[the remark this answers is not in the document]
Comment (oskarth) [reply examples/v2/basic2.nim:37]: Added todo inline

**Review by kdeme — APPROVED** [review]

**Pull request merged**